components/Testimonials.js

```diff
@@ -28,12 +28,12 @@ export default function Testimonials({ data }) {
                     />
                     <div>
                       <Link
-                        className="font-semibold text-gray-900"
+                        className="font-semibold text-primary-high dark:text-gray-300"
                         href={`/${testimonial.username}`}
                       >
                         {testimonial.name} ({`@${testimonial.username}`})
                       </Link>
-                      <div className="text-gray-600">{testimonial.bio}</div>
+                      <div className="text-gray-600 dark:text-gray-400">{testimonial.bio}</div>
```

Comment:
This needs changing to one of the new custom classes `text-gray-600 dark:text-gray-400`